### PR TITLE
Add People's Daily NER dataset

### PR DESCRIPTION
**Pull request by JetRunner**

### CODE REVIEW

File: datasets/peoples_daily_ner/peoples_daily_ner.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+# limitations under the License.
+
+# Lint as: python3
+"""Introduction to People's Daily Dataset"""
+
+import logging
+
+import datasets
+
+
```
Comment (lhoestq) [inline datasets/peoples_daily_ner/peoples_daily_ner.py:23]: is there a citation available somewhere for this dataset ?

**Review by lhoestq — COMMENTED** [review]
LGTM thanks :)

Before we merge, could you add a dataset card ? see here for more info: https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

Note that only the tags at the top of the dataset card are mandatory, if you feel like it's going to take too much time writing the rest to fill it all you can just skip the paragraphs

**Comment by JetRunner** [comment]
> LGTM thanks :)
> 
> 
> 
> Before we merge, could you add a dataset card ? see here for more info: https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card
> 
> 
> 
> Note that only the tags at the top of the dataset card are mandatory, if you feel like it's going to take too much time writing the rest to fill it all you can just skip the paragraphs

Nope. I don't think there is a citation. Also, can I do the dataset card later (maybe in bulk)?

**Comment by lhoestq** [comment]
We're doing one PR = one dataset to keep track of things. Feel free to add the tags later in this PR if you want to.
Also only the tags are required now, because we don't want people spending too much time on the cards

**Comment by JetRunner** [comment]
added @lhoestq

**Review by lhoestq — APPROVED** [review]
LGTM thank you !

**Comment by lhoestq** [comment]
Merging since the CI is fixed on master

**Pull request merged**